homeworks/pom.xml

```diff
@@ -49,9 +49,9 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
```

Comment:
All these dependencies are redundant, because you do not use them or they are included into spring-boot-starter-web, as for jdbc you are not using it at all, as well as jackson